pkgs/by-name/wh/whatsie/package.nix

```
@@ -20,6 +21,14 @@ stdenv.mkDerivation (finalAttrs: {
 
   sourceRoot = "${finalAttrs.src.name}/src";
 
+  desktopItem = makeDesktopItem {
+    name = finalAttrs.pname;
+    desktopName = "Whatsie";
+    icon = finalAttrs.pname;
+    exec = finalAttrs.pname;
+    comment = finalAttrs.meta.description;
+  };
```

```suggestion
  desktopItems = [ (makeDesktopItem {
    name = finalAttrs.pname;
    desktopName = "Whatsie";
    icon = finalAttrs.pname;
    exec = finalAttrs.pname;
    comment = finalAttrs.meta.description;
  }) ];
  
  nativeBuildInputs = [ copyDesktopItems ];
```

(complete input and apply format)